packages/nds-stacked-nav/stacked-nav.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module "@nice-digital/nds-stacked-nav" {
+	import React = require("react");
+
+	export interface StackedNavLinkProps {
+		destination?: string;
+		isCurrent?: boolean;
+		label?: React.ReactNode;
```

Comment:
Becasuse we're expecting label OR children we can probably do something clever with a pipe and alernative options with TS to enforce this